src/RestClient/endpoints/V3/Themes/ThemeConfigurations.ts

```diff
@@ -0,0 +1,36 @@
+import { AxiosInstance } from 'axios';
+
+import { themesPath } from './Themes';
+import type { ThemeConfiguration } from './types';
+
+class ThemeConfigurations {
+  private client: AxiosInstance;
+
+  constructor(client: AxiosInstance) {
+    this.client = client;
+  }
+
+  /**
+   * Returns a list of theme configurations
+   *
+   * @param uuid A valid Theme UUID
+   * @param params Query paramaters used to filter response
+   * @returns Promise resolving to a list of customer addresses
```

Comment:
Copy/paste - need to update to reflect what this method does

Reply:
Updated to theme info